packaging/Description.txt

```diff
@@ -0,0 +1,12 @@
+ Halide is a programming language designed to make it easier to write
+ high-performance image and array processing code on modern machines. Halide
+ currently targets:
+
+ * CPU architectures: X86, ARM, MIPS, Hexagon, PowerPC, RISC-V
```

Comment:
We should probably add WebAssembly to this list.

Reply:
Sure. I just copied it from the website.